### PR TITLE
Update SRG-OS-000080-GPOS-00048 for RHEL 9 STIG

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:
* Rework OCIL to be more aligned with actual OVAL check
* add fixtext to rules relevant to this SRG
* minor fixes

For more details, please read commit messages of all commits.

#### Rationale:
RHEL 9 STIG

### CODE REVIEW

**Review by Mab879 — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/bootloader-grub2/non-uefi/grub2_password/rule.yml [hunk 1]
```diff
@@ -114,3 +99,20 @@ warnings:
...
+    Enter password:
+    Confirm password:
+
+    Edit the /etc/grub.d/40_custom file and add or modify the following lines in the "### BEGIN /etc/grub.d/01_users ###" section:
+
+    set superusers="[someuniquestringhere]"
+    export superusers
+
+    Once the superuser account has been added, update the grub.cfg file by running:
+    $ sudo grub2-mkconfig -o {{{ grub2_uefi_boot_path }}}/grub.cfg
```
[the remark this answers is not in the document]
Comment (jan-cerny) [reply linux_os/guide/system/bootloader-grub2/non-uefi/grub2_password/rule.yml:118]: Yes!

**Comment by jan-cerny** [comment]
After #8628 has been merged I have rebased this PR on master, and extended the macro grub_command and used it in the rules modified in this PR.

**Comment by jan-cerny** [comment]
rebase it again

File: linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_admin_username/rule.yml [hunk 2]
```diff
@@ -80,3 +80,17 @@ warnings:
         <tt>grub.cfg</tt> file as the grub2-mkconfig command overwrites this file.
 
 platform: machine
+
+fixtext: |-
+    Configure the system to require a grub bootloader password for the grub superuser account.
```
Comment (Mab879) [inline linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_admin_username/rule.yml:85]:
```suggestion
    Configure the system to have a unique username for the grub superuser account.
```

**Review by Mab879 — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_admin_username/rule.yml [hunk 3]
```diff
@@ -80,3 +80,17 @@ warnings:
         <tt>grub.cfg</tt> file as the grub2-mkconfig command overwrites this file.
 
 platform: machine
+
+fixtext: |-
+    Configure the system to have a unique username for the grub superuser account.
```
Comment (Mab879) [inline linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_admin_username/rule.yml:85]:
```suggestion
    Configure {{{ full_name }}} to have a unique username for the grub superuser account.
```

File: linux_os/guide/system/bootloader-grub2/non-uefi/grub2_password/rule.yml [hunk 4]
```diff
@@ -114,3 +95,21 @@ warnings:
         <tt>grub.cfg</tt> file as the grub2-mkconfig command overwrites this file.
 
 platform: machine
+
+fixtext: |-
+    Configure the system to require a grub bootloader password for the grub superuser account.
```
Comment (Mab879) [inline linux_os/guide/system/bootloader-grub2/non-uefi/grub2_password/rule.yml:100]:
```suggestion
    Configure {{{ full_name }}} to require a grub bootloader password for the grub superuser account.
```

**Review by Mab879 — CHANGES_REQUESTED** [review]

**Comment by jan-cerny** [comment]
The operating system is now replaced by full_name.

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
@jan-cerny The errors from content test filtering concerning me. 

@ggbecker @mildas Any thoughts?

**Comment by jan-cerny** [comment]
I have rebased the PR on the top of the current upstream master.

**Review by Mab879 — APPROVED** [review]

**Pull request merged**